tests/components/monoprice/test_media_player.py

```diff
@@ -305,6 +319,15 @@ def restore_zone(self, zone):
     assert ["one", "three"] == state.attributes[ATTR_INPUT_SOURCE_LIST]
 
 
+async def test_source_list_with_options(hass):
+    """Test source list property."""
+    await _setup_monoprice_with_options(hass, MockMonoprice())
+
+    state = hass.states.get(ZONE_1_ID)
+    # Note, the list is sorted!
+    assert ["two", "four"] == state.attributes[ATTR_INPUT_SOURCE_LIST]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5232399)